package/python-kiwi-spec-template

```diff
@@ -291,10 +291,14 @@ Provides:       kiwi-filesystem:ext3
 Provides:       kiwi-filesystem:ext4
 Provides:       kiwi-filesystem:squashfs
 Provides:       kiwi-filesystem:xfs
+Provides:       kiwi-filesystem:erofs
```

Comment:
This also needs `%if ! (0%{?suse_version} && 0%{?suse_version} < 1600)` otherwise OBS will happily let you run a build guaranteed to fail.

Reply:
yes I did the same mistake again, will put it into the same condition block

Reply:
done